awx/ui_next/src/components/LoadingSpinner/LoadingSpinner.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { Spinner as PFSpinner } from '@patternfly/react-core';
+import styled from 'styled-components';
+
+const Spinner = styled(PFSpinner)`
```

Comment:
nit-picky, but it makes more sense to me to list the container (`UpdatingContent`) above the spinner